docs/install/README.md

```diff
@@ -371,38 +373,451 @@ Status:
+
+```console
+kubectl -n k8ssandra-operator get deployment
+NAME                                    READY   UP-TO-DATE   AVAILABLE   AGE
+k8ssandra-operator-cass-operator        1/1     1            1           85s
+k8ssandra-operator-k8ssandra-operator   1/1     1            1           85s
+```
+
+Verify that the `K8SSANDRA_CONTROL_PLANE` environment variable is set to `false`:
+
+```console
+kubectl get deployment k8ssandra-operator-k8ssandra-operator -o jsonpath='{.spec.template.
+spec.containers[0].env[?(@.name=="K8SSANDRA_CONTROL_PLANE")].value}'
+```
+
+#### Create a ClientConfig
+Now we need to create a `ClientConfig` for the `kind-k8ssandra-1` cluster. We will use 
+the `create-clientconfig.sh` script which can be found [here](https://github.com/k8ssandra/k8ssandra-operator/blob/main/scripts/create-clientconfig.sh).
+
+Here is a summary of what the script does:
 
-## Single Cluster Install with Kustomize
-We will first look at a single cluster install to demonstrate that while K8ssandra Operator is designed for multi-cluster use, it can be used in a single cluster without any extra configuration.
+* Get the k8ssandra-operator service account from the data plane cluster
+* Extract the service account token
+* Extract the CA cert
+* Create a kubeonfig using the token and cert
+* Create a secret for the kubeconfig in the control plane cluster
+* Create a ClientConfig in the control plane cluster that references the secret
 
-### Create kind cluster
+Create a `ClientConfig` in the `kind-k8ssandra-0` cluster using the service account 
+token and CA cert from `kind-k8ssandra-1`:
+
+```sh
+./create-clientconfig.sh --namespace k8ssandra-operator --src-kubeconfig build/kubeconfigs/k8ssandra-1.yaml --dest-kubeconfig build/kubeconfigs/k8ssandra-0.yaml --in-cluster-kubeconfig build/kubeconfigs/updated/k8ssandra-1.yaml --output-dir clientconfig
+```
+The script stores all the artifacts that it generates in a directory which is specified with the `--output-dir` option. If not specified, a temp directory is created.
+
+You can specify the namespace where the secret and ClientConfig are created with the `--namespace` option.
+
+The `--in-cluster-kubeconfig` option is required for clusters that run locally like kind.
+
+#### Restart the control plane
+**TODO:** Add reference to ticket explaining the need to restart.
+
+Make the active context `kind-k8ssandra-0`:
+
+```console
+kubectx kind-k8ssandra-0
+```
+
+Restart the operator:
+
+```console
+kubectl -n k8ssandra-operator rollout restart deployment k8ssandra-operator
+```
+
+#### Deploy a K8ssandraCluster
+Now we will create a K8ssandraCluster that consists of a Cassandra cluster with 2 DCs and 3 
+nodes per DC, and a Stargate node per DC.
+
+```sh
+cat <<EOF | kubectl apply -f -
+apiVersion: k8ssandra.io/v1alpha1
+kind: K8ssandraCluster
+metadata:
+  name: demo
+spec:
+  cassandra:
+    cluster: demo
+    serverVersion: "3.11.11"
```

Comment:
I'd be in favor of using 4.0.0 as a default in our samples.

Reply:
I think that's a good idea as well... as long as 4.0.0 is currently working with the recent Stargate complicates that we've found anyhow?

Reply:
I had used 3.11.11 due to the Stargate schema problems. If it works, then I agree and will change it.

Reply:
Why are we not using 4.0.1?

Reply:
I tried an upgrade and one of the nodes failed to come back up.

The initial deployment with 4.0.0 worked as expected.

Reply:
I have updated the examples to use 4.0.1.